app/controllers/api/v2/users_controller.rb

```diff
@@ -36,21 +36,32 @@ def index
       def show
       end
 
+      def_param_group :user_params do
+        param :login, String, :required => true
+        param :firstname, String, :required => false
+        param :lastname, String, :required => false
+        param :mail, String, :required => true
+        param :description, String, :required => false
+        param :admin, :bool, :required => false, :desc => N_("is an admin account")
+        param :password, String, :required => true
+        param :default_location_id, Integer if SETTINGS[:locations_enabled]
+        param :default_organization_id, Integer if SETTINGS[:organizations_enabled]
+        param :auth_source_id, Integer, :required => true
+        param :timezone, ActiveSupport::TimeZone.all.map(&:name), :required => false, :desc => N_("User's timezone")
+        param :locale, FastGettext.available_locales, :required => false, :desc => N_("User's preferred locale")
+        param_group :taxonomies, ::Api::V2::BaseController
+      end
+
       def_param_group :user do
         param :user, Hash, :required => true, :action_aware => true do
-          param :login, String, :required => true
-          param :firstname, String, :required => false
-          param :lastname, String, :required => false
-          param :mail, String, :required => true
-          param :description, String, :required => false
-          param :admin, :bool, :required => false, :desc => N_("is an admin account")
-          param :password, String, :required => true
-          param :default_location_id, Integer if SETTINGS[:locations_enabled]
-          param :default_organization_id, Integer if SETTINGS[:organizations_enabled]
-          param :auth_source_id, Integer, :required => true
-          param :timezone, ActiveSupport::TimeZone.all.map(&:name), :required => false, :desc => N_("User's timezone")
-          param :locale, FastGettext.available_locales, :required => false, :desc => N_("User's preferred locale")
-          param_group :taxonomies, ::Api::V2::BaseController
+          param_group :user_params
+        end
+      end
+
+      def_param_group :current_user_update do
+        param :user, Hash, :required => true, :action_aware => true do
+          param_group :user_params
+          param :current_password, String, :required => true
```

Comment:
Also a description when this field is required would be handy. Could you add it, please?